src/Keboola/DbExtractor/Extractor/Extractor.php

```diff
@@ -177,7 +177,7 @@ public function export(array $table): array
             $this->logger,
             $maxTries,
             RetryProxy::DEFAULT_BACKOFF_INTERVAL,
-            ['Keboola\DbExtractor\Exception\DeadConnectionException']
+            ['Keboola\DbExtractor\Exception\DeadConnectionException', 'ErrorException']
```

Comment:
in second case it doesn't need a namespace?

Reply:
sorry, I mean full path to class

Reply:
ErrorException is the full path.  see: https://github.com/keboola/db-extractor-common/blob/master/src/Keboola/DbExtractor/RetryProxy.php#L21

Reply:
better use `DeadConnectionException::class` and `ErrorException::class` ?

Reply:
ok then :) thx

Reply:
ok, updating...

Reply:
done, I'll merge when the tests pass